src/transform/ElementUtilities.js

```diff
@@ -1,4 +1,21 @@
 /**
+ * Polyfill function that tells whether a given element matches a selector.
+ * @param {!Element} el Element
+ * @param {!string} selector Selector to look for
+ * @returns {boolean} Whether the element matches the selector
+ */
+const matchesSelectorCompat = (el, selector) => {
```

Comment:
Oh this is great! 

The only thing I'm wondering about is the naming... does `safeMatches` sound any better/worse?